conf/base.config

```diff
@@ -74,13 +74,18 @@ process {
     }
 
     withName: MEGAN_RMA2INFO_TSV {
-        cpus   = { check_max( 1                  , 'cpus'    ) }
-        memory = { check_max( 6.GB * task.attempt, 'memory'  ) }
-        time   = { check_max( 4.h  * task.attempt, 'time'    ) }
+        cpus   = { check_max( 1                   , 'cpus'    ) }
+        memory = { check_max( 6.GB  * task.attempt, 'memory'  ) }
+        time   = { check_max( 4.h   * task.attempt, 'time'    ) }
     }
     withName: MEGAN_RMA2INFO_KRONA {
-        cpus   = { check_max( 1                  , 'cpus'    ) }
-        memory = { check_max( 6.GB * task.attempt, 'memory'  ) }
-        time   = { check_max( 4.h  * task.attempt, 'time'    ) }
+        cpus   = { check_max( 1                   , 'cpus'    ) }
+        memory = { check_max( 6.GB  * task.attempt, 'memory'  ) }
+        time   = { check_max( 4.h   * task.attempt, 'time'    ) }
+    }
+    withName: FALCO {
+        cpus   = { check_max( 6                   , 'cpus'    ) }
+        memory = { check_max( 36.GB * task.attempt, 'memory'  ) }
```

Comment:
I think this is overkill! 1GB to 36GB is too much as it's actually small job (just needs a bit more than fastqC) , maybe increase it to 4.GB

Reply:
Ok. I will change the memory to 4GB. Should I also keep the CPU as the original "1"?

Reply:
That still seems high... and FastQC being a medium process is a bit mof a misnomer, increasing threads for FastQC is only to increase the memory (1 CPU == 500 MB RAM... don't ask me why...).

Falco is still multithreaded so keep the CPUs :)

Reply:
Ok. I will change the memory to 4GB and keep the cpu 6.

Reply:
@jfy133 According to this paper, Falco is known for its memory efficiency, and allocating 4GB of memory would indeed be a reasonable number. Additionally, Falco has been reported to utilize approximately 14 times less memory than FastQC when processing long-reads. 
So the chosen memory setting of 4GB appears to be well-suited for running Falco efficiently and effectively for your analysis.

https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7845152/

If you think it's fine, can you approve it?